WhaleSpotting/Models/Enums/OrcaPod.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace WhaleSpotting.Models.Enums
+{
+    public enum OrcaPod
+    {
+        j, 
```

Comment:
Do we not know what these are? (or are these the actual pods?)

Reply:
We discussed this and changed to list<OrcaPod>.